INFO 2019-09-09 11:05:28,595 basehttp 124 "GET / HTTP/1.1" 200 1714
WARNING 2019-09-09 11:05:28,704 base 152 Not Found: /favicon.ico
WARNING 2019-09-09 11:05:28,709 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 1967
WARNING 2019-09-10 14:10:56,402 base 152 Not Found: /
WARNING 2019-09-10 14:10:56,408 basehttp 124 "GET / HTTP/1.1" 404 2106
WARNING 2019-09-10 14:10:57,249 base 152 Not Found: /favicon.ico
WARNING 2019-09-10 14:10:57,266 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2157
WARNING 2019-09-10 14:12:16,081 base 152 Not Found: /
WARNING 2019-09-10 14:12:16,085 basehttp 124 "GET / HTTP/1.1" 404 2106
WARNING 2019-09-10 14:12:16,423 base 152 Not Found: /favicon.ico
WARNING 2019-09-10 14:12:16,423 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2157
WARNING 2019-09-10 14:13:04,990 base 152 Not Found: /
WARNING 2019-09-10 14:13:04,991 basehttp 124 "GET / HTTP/1.1" 404 2106
WARNING 2019-09-10 14:13:05,340 base 152 Not Found: /favicon.ico
WARNING 2019-09-10 14:13:05,343 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2157
WARNING 2019-09-10 14:20:02,602 base 152 Not Found: /sms_code/13789898989
WARNING 2019-09-10 14:20:02,606 basehttp 124 "GET /sms_code/13789898989 HTTP/1.1" 404 2184
INFO 2019-09-10 14:20:32,250 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
ERROR 2019-09-10 14:20:32,266 exception 135 Internal Server Error: /sms_codes/13789898989/
Traceback (most recent call last):
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/changle-lab/Desktop/shopping_mall/shoppingMall/meiduo_mall/meiduo_mall/apps/users/views.py", line 23, in get
    conn = get_redis_connection('sms_code')
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django_redis/__init__.py", line 14, in get_redis_connection
    cache = caches[alias]
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/cache/__init__.py", line 77, in __getitem__
    "Could not find config for '%s' in settings.CACHES" % alias
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'sms_code' in settings.CACHES
ERROR 2019-09-10 14:20:32,329 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 500 18396
INFO 2019-09-10 14:20:51,903 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
ERROR 2019-09-10 14:20:51,952 exception 135 Internal Server Error: /sms_codes/13789898989/
Traceback (most recent call last):
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/changle-lab/Desktop/shopping_mall/shoppingMall/meiduo_mall/meiduo_mall/apps/users/views.py", line 23, in get
    conn = get_redis_connection('sms_code')
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django_redis/__init__.py", line 14, in get_redis_connection
    cache = caches[alias]
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/cache/__init__.py", line 77, in __getitem__
    "Could not find config for '%s' in settings.CACHES" % alias
django.core.cache.backends.base.InvalidCacheBackendError: Could not find config for 'sms_code' in settings.CACHES
ERROR 2019-09-10 14:20:52,003 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 500 18396
INFO 2019-09-10 14:26:04,895 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
ERROR 2019-09-10 14:26:04,912 exception 135 Internal Server Error: /sms_codes/13789898989/
Traceback (most recent call last):
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/changle-lab/.virtualenvs/shoppingMall/lib/python3.6/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/changle-lab/Desktop/shopping_mall/shoppingMall/meiduo_mall/meiduo_mall/apps/users/views.py", line 26, in get
    ccp = CCP()
  File "/home/changle-lab/Desktop/shopping_mall/shoppingMall/meiduo_mall/meiduo_mall/apps/libs/yuntongxun/sms.py", line 53, in __new__
    cls._instance.rest = REST(_serverIP, _serverPort, _softVersion)
NameError: name 'REST' is not defined
ERROR 2019-09-10 14:26:04,946 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 500 18282
INFO 2019-09-10 14:30:24,356 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 14:30:24,684 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 14:33:44,025 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 14:33:44,313 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:20,740 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:20,761 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:21,699 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:21,706 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:23,265 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:23,278 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:23,692 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:23,700 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:24,256 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:24,262 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:24,778 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:24,786 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:25,206 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:25,217 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:25,675 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:25,693 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:26,077 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:26,089 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:26,499 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:26,504 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:50:27,569 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:50:27,572 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:57:28,269 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 15:57:28,559 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
INFO 2019-09-10 15:57:32,652 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
WARNING 2019-09-10 15:57:32,671 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 403 30
INFO 2019-09-10 17:35:16,220 basehttp 124 "GET /sms_codes/13789898989 HTTP/1.1" 301 0
INFO 2019-09-10 17:35:16,330 basehttp 124 "GET /sms_codes/13789898989/ HTTP/1.1" 200 4
WARNING 2019-09-10 22:07:57,526 base 152 Not Found: /mobiles/17310115251/count/
WARNING 2019-09-10 22:07:57,530 basehttp 124 "GET /mobiles/17310115251/count/ HTTP/1.1" 404 2208
WARNING 2019-09-10 22:07:57,643 base 152 Not Found: /mobiles/17310115251/count/
WARNING 2019-09-10 22:07:57,645 basehttp 124 "GET /mobiles/17310115251/count/ HTTP/1.1" 404 2208
INFO 2019-09-10 22:07:57,713 basehttp 124 "GET /sms_codes/17310115251/ HTTP/1.1" 200 4
WARNING 2019-09-10 22:57:57,009 base 152 Not Found: /mobiles/17310115251/count/
WARNING 2019-09-10 22:57:57,013 basehttp 124 "GET /mobiles/17310115251/count/ HTTP/1.1" 404 2208
WARNING 2019-09-10 22:57:57,172 base 152 Not Found: /mobiles/17310115251/count/
WARNING 2019-09-10 22:57:57,174 basehttp 124 "GET /mobiles/17310115251/count/ HTTP/1.1" 404 2208
INFO 2019-09-10 22:57:57,224 basehttp 124 "GET /sms_codes/17310115251/ HTTP/1.1" 200 4
WARNING 2019-09-10 23:17:47,045 base 152 Not Found: /
WARNING 2019-09-10 23:17:47,046 basehttp 124 "GET / HTTP/1.1" 404 2106
WARNING 2019-09-10 23:17:47,178 base 152 Not Found: /favicon.ico
WARNING 2019-09-10 23:17:47,180 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2157
WARNING 2019-09-10 23:18:19,345 base 152 Not Found: /
WARNING 2019-09-10 23:18:19,350 basehttp 124 "GET / HTTP/1.1" 404 2112
WARNING 2019-09-10 23:18:19,448 base 152 Not Found: /favicon.ico
WARNING 2019-09-10 23:18:19,450 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2163
WARNING 2019-09-10 23:18:31,139 base 152 Not Found: /
WARNING 2019-09-10 23:18:31,149 basehttp 124 "GET / HTTP/1.1" 404 2112
ERROR 2019-09-10 23:19:10,819 exception 80 Invalid HTTP_HOST header: 'www.meiduo.site:8000'. You may need to add 'www.meiduo.site' to ALLOWED_HOSTS.
WARNING 2019-09-10 23:19:10,935 basehttp 124 "GET / HTTP/1.1" 400 71720
ERROR 2019-09-10 23:19:11,157 exception 80 Invalid HTTP_HOST header: 'www.meiduo.site:8000'. You may need to add 'www.meiduo.site' to ALLOWED_HOSTS.
WARNING 2019-09-10 23:19:11,211 basehttp 124 "GET /favicon.ico HTTP/1.1" 400 71771
WARNING 2019-09-10 23:20:05,062 base 152 Not Found: /usernames/ffdsffasfasd/count/
WARNING 2019-09-10 23:20:05,063 basehttp 124 "GET /usernames/ffdsffasfasd/count/ HTTP/1.1" 404 2217
ERROR 2019-09-10 23:22:12,808 exception 80 Invalid HTTP_HOST header: 'api.meiduo.site:8000'. You may need to add 'api.meiduo.site' to ALLOWED_HOSTS.
WARNING 2019-09-10 23:22:12,867 basehttp 124 "GET /usernames/ffffff/count/ HTTP/1.1" 400 72317
WARNING 2019-09-10 23:24:02,394 base 152 Not Found: /usernames/grgggg/count/
WARNING 2019-09-10 23:24:02,405 basehttp 124 "GET /usernames/grgggg/count/ HTTP/1.1" 404 2199
